Apps/System/systemapp.c

```diff
@@ -184,14 +182,9 @@ static void about_window_load(Window *window)
     layer_add_child(window_layer, scroll_layer_get_layer(s_about_scroll));
     layer_add_child(window_layer, status_bar_layer_get_layer(status_bar));
 
-    #ifdef PBL_BW
-         //TODO: Get black and white rocket bitmap for Classic
-        logo_bitmap = gbitmap_create_with_resource(RESOURCE_ID_REBBLE_LOGO_BW);
-        rocket_bitmap = gbitmap_create_with_resource(RESOURCE_ID_TO_MOON_BW);       
-    #else
-        logo_bitmap = gbitmap_create_with_resource(RESOURCE_ID_REBBLE_LOGO_DARK);
-        rocket_bitmap = gbitmap_create_with_resource(RESOURCE_ID_TO_MOON);
-    #endif
+    //TODO: Get black and white rocket bitmap for Classic
+    logo_bitmap = gbitmap_create_with_resource(PBL_IF_BW_ELSE(RESOURCE_ID_REBBLE_LOGO_BW, RESOURCE_ID_REBBLE_LOGO_DARK));
```

Comment:
Maybe another idea is to just give it the same resource name in the .json file on both platforms?